GPU/GPUTracking/Merger/GPUTPCGMTrackParam.h

```diff
@@ -224,12 +225,13 @@ class GPUTPCGMTrackParam
   GPUd() bool FollowCircleChk(float lrFactor, float toY, float toX, bool up, bool right);
   GPUd() int initResetT0();
 
-  float mX;        // x position
-  float mTZOffset; // Z offset with early transform, T offset otherwise
-  float mP[5];     // 'active' track parameters: Y, Z, SinPhi, DzDs, q/Pt
-  float mC[15];    // the covariance matrix for Y,Z,SinPhi,..
-  float mChi2;     // the chi^2 value
-  int mNDF;        // the Number of Degrees of Freedom
+  float mX;                           // x position
+  float mTZOffset;                    // Z offset with early transform, T offset otherwise
+  float mP[5];                        // 'active' track parameters: Y, Z, SinPhi, DzDs, q/Pt
+  float mC[15];                       // the covariance matrix for Y,Z,SinPhi,..
+  float mChi2;                        // the chi^2 value
+  int mNDF;                           // the Number of Degrees of Freedom
+  o2::utils::DebugStreamer mStreamer; ///<! streamer for writing out track update
```

Comment:
Instead of add a DebugStreamer data member everywhere, can't we do something like
`static o2::utils::DebugStreamer streamer("debug_InterpolateReject", "UPDATE")`
in the place where we use them? In that way, we don't need to include the DebugStreamer header in all the header data files? This could be done in the way I do it for my GPUROOTDump, which  has very similar functionality, e.g. https://github.com/davidrohr/AliceO2/blob/cf859182f75967b35c46bc643ef44222ddd284c5/GPU/GPUTracking/qa/GPUQA.cxx#L1716

Reply:
Thanks, I will check this. Would be indeed better if this works...

Reply:
It is not possible in this way. We use **local variables** that you cannot access in the header. **The ability to stream local variables is the main advantage of the streaming approach.**
This is essential for understanding the algorithm and the data itself.

Reply:
But why would you need to access something in the header? My point was to just move the mStreamer data member of the class to a local static variable. Since you anyway call setStreamer before you do the actual streaming, I don't see why you need the mStreamer as a data member.

Reply:
If you can make guarantee the same logging functionality (including local variables) then it is OK for me. In my previous respond, I assumed in your solution it will be not possible.